jobs/mongodb_migration/src/mongodb_migration/main.py

```diff
@@ -11,25 +12,49 @@
 from mongodb_migration.plan import Plan
 from mongodb_migration.resources import MigrationsMongoResource
 
-if __name__ == "__main__":
+
+def run_job() -> None:
     job_config = JobConfig.from_env()
 
     init_logging(log_level=job_config.common.log_level)
     # ^ set first to have logs as soon as possible
 
     with (
-        CacheMongoResource(database=job_config.cache.mongo_database, host=job_config.cache.mongo_url),
-        QueueMongoResource(database=job_config.queue.mongo_database, host=job_config.queue.mongo_url),
+        CacheMongoResource(
+            database=job_config.cache.mongo_database, host=job_config.cache.mongo_url
+        ) as cache_resource,
+        QueueMongoResource(
+            database=job_config.queue.mongo_database, host=job_config.queue.mongo_url
+        ) as queue_resource,
         MigrationsMongoResource(
             database=job_config.database_migrations.mongo_database, host=job_config.database_migrations.mongo_url
-        ),
+        ) as migrations_database_resource,
     ):
+        if cache_resource.check() is False:
+            logging.info(
```

Comment:
Not sure if it should be an info, maybe Warning since it will avoid running the migrations

Reply:
OK

Reply:
Done